tests/sentry/snuba/metrics/test_extraction.py

```diff
@@ -17,6 +18,15 @@
 from sentry.testutils.pytest.fixtures import django_db_all
 
 
+@django_db_all
+def test_equality_of_specs(default_project) -> None:
+    spec_1 = OnDemandMetricSpec("count()", "issue:FOO", groupbys=["release", "country_code"])
+    spec_2 = OnDemandMetricSpec("count()", "issue:FOO", groupbys=["country_code", "release"])
+    assert are_specs_equal(
```

Comment:
This is basically emulating the equality check that we added on the previous PR for `_merge_metric_specs`:
https://github.com/getsentry/sentry/blob/6f3a2e4a54a2efc26534c2d9ab86a3e74eec6fec/src/sentry/relay/config/metric_extraction.py#L263

Calling `_merge_metric_specs` is done via `get_metric_extraction_config`.